doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-section.adoc

```diff
@@ -16,36 +16,156 @@ After login, users may upload dashboard files exported from {CENTRAL}
+. In *Export Wizard* panel, verify the pages that you want to include in the ZIP file and click *Next*.
+. Select *Export Wizard* and click *Open*.
++
+You can see the dashbuilder runtime home page. If you are not logged in, you are redirected to the login page.
+. Log in to Dashbuilder Runtime.
+. Go to *Dashboards → Runtime Dashboards* and select the exported ZIP file.
++
+The selected data is exported and dashbuilder runtime updates the model content when it is opened.
 
-image::BAM/GradualExport.png[]
 
 
 == Execution modes
 Dashbuilder Runtime has execution modes, which are:
 
-* *SINGLE*: This is the default mode. When a dashboard is uploaded then it will be used in Runtime. It is also possible to refer on dashboards under the directory configured using the system property `dashbuilder.import.base.dir`, default value is `/tmp/dashbuilder/models`. 
+* *SINGLE*: This is the default mode. When a dashboard is uploaded then it will be used in Runtime. It is also possible to refer on dashboards under the directory configured using the system property `dashbuilder.import.base.dir`, default value is `/tmp/dashbuilder/models`.
 If a file is in the `models` directory, then it is possible to load it using `RUNTIME_URL?import=FILE_NAME` in the URL. For example, the file `sales_dashboard.zip`, placed in the `models` directory, can be loaded using `RUNTIME_URL?import=sales_dashboard`(notice that the file extension is not necessary).
-* *STATIC*: In this mode Dashbuilder Runtime will not prompt users for upload and use the model file configured using system property `dashbuilder.runtime.import`. 
+* *STATIC*: In this mode Dashbuilder Runtime will not prompt users for upload and use the model file configured using system property `dashbuilder.runtime.import`.
 * *MULTI*:  Finally in multi mode users can upload multiple dashboards in the same Dashbuilder Runtime installation. To enable it the system property `dashbuilder.runtime.multi` must be set as true.
 
 image::BAM/DashbuilderRuntimeMultiMode.png[]
 
 == Embedding dashboards
 
-Dashboards running in Dashbuilder Runtime can be embedded in third-party applications using an `iframe`. To embed dashboards, the query parameter *standalone* should be added to the URL along with the dashboard page name using the query param *perspective*: 
+Dashboards running in Dashbuilder Runtime can be embedded in third-party applications using an `iframe`. To embed dashboards, the query parameter *standalone* should be added to the URL along with the dashboard page name using the query param *perspective*:
 
 `RUNTIME_URL?standalone&perspective=PAGE_NAME`
 
-In multi it is also required to specify the import id:
+In case of multi mode, it is also required to specify the import id:
 
 `RUNTIME_URL?standalone&perspective=PAGE_NAME&import=IMPORT_NAME`
 
+In case of multi mode, you must specify which dashboard you want to use by using the `import={dashboard_name}` property:
```

Comment:
```suggestion
In case of multi-mode, you must specify which dashboard you want to use by using the `import={dashboard_name}` property, for example:
```